c++/testprogram/testProgram.cpp

```diff
@@ -141,6 +141,7 @@ subTaskUsingSharedResource(int taskNbr, int iterNbr)
 void
 controlTask(int groupNbr, const char* groupName, int durationMultipler)
 {
+    plFunction();
```

Comment:
I have 2 remarks for this particular addition:
 1) As said in https://dfeneyrou.github.io/palanteer/instrumentation_api_cpp.md.html#c++instrumentationapi/structurelogging/plfunction  , some not so old compilers do not support plFunction() because  __FUNCTION__ is not considered as constexpr, so plFunctionDyn() shall be used instead (I would have been glad to make the "Dyn" stuff transparent, but it was not possible due to constexpr parameters not a thing event in C++20).
  As this is an example program it is better to keep compatibility even for those compilers.
 2) it will create a steady scope for the full thread duration, so without any "hole" up to the root. That is perfectly ok, but I wanted to let such "holes" in this example (game frame rendering would let such holes between frames).

Reply:
Note that the instrumentation library has ~50 tests. 
But plFunction is not used in the example program due to the point above with "old compiler", that is why it was not caught.

Reply:
I'll remove the plFunction call.